deploy/olm-catalog/0.9.0/pmem-csi-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
I think this is not intentional to add files under 'deploy/olm-catalog', is it?

Reply:
Not intentional. `.gitignore` should have been updating when adding that directory to the built files.

Reply:
Ah, it was the other way around: I still had the directory, but 435f821906c055b8dfd6a918ba36dc3e2137dcea removed the exclude rule.